styles/modules/_form.scss

```diff
@@ -0,0 +1,42 @@
+.form {
+    &--compact {
+        @include mq($screen-s) {
+            display: flex;
+            justify-content: center;
+        }
+        .form__title {
+            line-height: $button-height;
+            margin-right: 20px;
```

Comment:
maybe we can use heading for normal form but for compact variant we can't because here it's like label more than title but need to be out of the input div